test/psse.jl

```diff
@@ -319,4 +319,18 @@ end
             end
         end
     end
+
+    @testset "import all" begin
+        data = PowerModels.parse_file("../test/data/pti/case30.raw"; import_all=true)
+
+        @test length(data) == 19
+        for (key, n) in zip(["bus", "load", "shunt", "gen", "branch", "dcline"], [30, 21, 2, 6, 41, 0])
+            @test length(data[key]) == n
+        end
```

Comment:
I would recommend adding 2-3 spot-checks that specific extended parameter names made it into the data dict of each component.

Reply:
ah, so what i meant to do was check the number of keys per item, like number of keys in each bus, but i made a mistake when i set up the test